vueui/src/components/view/turrets/turret-controls.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <div class="item">
+      Behaviour controls are {{locked ? "locked" : "unlocked"}}.
+    </div>
+    <div v-if="access">
+      <div class="item">
+        <div class="itemLabelWide">
+          Turret Status:
+        </div>
+        <div class="itemContentNarrow">
+          <vui-button :params="{'command' : 'enable', 'value' : 1}" :class="{'redButton': enabled}">Enabled</vui-button>
+          <vui-button :params="{'command' : 'enable', 'value' : 0}" :class="{'selected': !enabled}">Disalbed</vui-button>
+        </div>
+      </div>	
+
+      <div v-if="is_lethal">
+        <div class="item">
+          <div class="itemLabelWide">
+            Lethal Mode:
+          </div>
+          <div v-if="can_switch">
+            <div class="itemContentNarrow">
+              <vui-button :params="{'command' : 'lethal', 'value' : 1}" :class="{'redButton': lethal}">On</vui-button>
+              <vui-button :params="{'command' : 'lethal', 'value' : 0}" :class="{'selected': !lethal}">Off</vui-button>
+            </div>
+          </div>
+
+          <div v-if-else>
```

Comment:
```suggestion
          <div v-else>
```
EDIT: Err actually this v-if / v-else is not needed. Look at comment above.